Doc/library/multiprocessing.rst

```diff
@@ -1187,6 +1187,81 @@ For example:
     the data in the pipe is likely to become corrupted, because it may become
     impossible to be sure where the message boundaries lie.
 
+Custom Reduction
+~~~~~~~~~~~~~~~~
```

Comment:
You'll need some `versionadded` directive at some point.

Reply:
Should we add this? Technically this PR is fixing the previous implementation, although as the old one was broken, one could argue that we are adding the feature.